cmd/forwarder/fixer.go

```diff
@@ -49,9 +49,9 @@ func getMetadata(req *http.Request, cred *credential, metadataId string) ([]byte
 				metadataWriter.WriteString(metadataId)
 				foundMetadata = true
 			}
-			metadataWriter.WriteString(" fields=\"{'credential_deprecated': true, 'credential_name': '")
+			metadataWriter.WriteString(" fields=\"credential_deprecated=true,credential_name=")
```

Comment:
Suggestion:

```go
metadataWriter.WriteString(` fields="credential_deprecated=true,credential_name=`)
metadataWriter.WriteString(`"`)
```

Reply:
@freeformz is backticks just the usual way of doing strings in go?

Reply:
... or is it because that way this will require less escaping? :)